openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -126,6 +126,13 @@ def create_mock_profile(self, user):
         legacy_profile.phone_number = "+18005555555"
         legacy_profile.save()
 
+    def create_user_registration(self, user):
+        """
+        Helper method that creates a registration object for the specified user
+        :return:
```

Comment:
remove this return. Also, this return is not in format with openedX codebase docstring format. Ref: https://edx-developer-guide.readthedocs.io/en/latest/style_guides/python-guidelines.html#docstrings